src/components/Form/DeleteDocumentForm.jsx

```diff
@@ -49,6 +49,7 @@ const DeleteDocumentForm = () => {
     margin: '20px 0'
   };
 
+  /* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Probably will be fixed once we've refactored the forms from `<form>` to MUI `<FormControl>`